src/main/core/Event/CatalogEvents/ResourceEvents.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Claroline\CoreBundle\Event\CatalogEvents;
+
+final class ResourceEvents
+{
+    /**
+     * @Event("Claroline\CoreBundle\Event\Resource\LoadResourceEvent")
+     */
+    public const RESOURCE_OPEN = 'event.functional.resource_open';
```

Comment:
`event.functional` can be removed. It doesn't provide any information here. I will not put a comment on each one, but this is true for all the events naming.